utility/environment_check.py

```diff
@@ -44,46 +58,32 @@ def environment_checker(request):
     get_status_before_execution()
 
 
+def assign_get_values(env_status_dict, key, kind):
```

Comment:
Please add missing docsting to the functions in this file

Reply:
done